### ISSUE TITLE
Simple model quality stats

### ISSUE THREAD

**Issue by jboelaert** [emilienschultz/activetigger#144]
It would be very nice to have quality stats for the simple model, maybe in a new subtab of Annotation tab.
The idea is to do 10-CV predictions on the tagged training data, and produce a confusion table and accuracy / F1 stats.
Also, simple training accuracy (or macro F1) on the training set (+OOB accuracy for random forest) would be nice.

**Comment by emilienschultz** [comment]
already on the tab in the top when the simplemodel is calculated no ?

**Comment by emilienschultz** [comment]
Made it nicer

**Comment by jboelaert** [comment]
I reopen this: it's actually another thing. 
Right now the displayed stats are the training stats, I would like to have 10-CV stats for the simple model.
This is useful for comparison with the BERT stats: if BERT doesn't do better than the simple model, you should probably annotate more.

**Comment by emilienschultz** [comment]
I made changes in the computation for statistics : now it is a specific function/model type, and I compute it for simplemodel. Will be available in the next update
